Urban-Green-Lab/application
# Create home screen and form

Fixes #33 

# Fetching Instructions
```shell
git fetch --all
git checkout add-home-screen
```

# Description
Created home screen and added form that adds user info to local storage

# List of changes
- Moved file locations for cleaner code
- Create home screen

# Checklist:

# To test
- Navigate to home screen
- complete form
- open application in dev tools
- check local storage for `user` should have the values you entered

![Screen Shot 2020-10-29 at 10 48 20 AM](https://user-images.githubusercontent.com/29741570/97597872-4ff2b780-19d4-11eb-97f3-8f6aa9f990e2.png)
